src/lib/Preview.js

```diff
@@ -987,6 +986,12 @@ class Preview extends EventEmitter {
         // Add the response interceptor to the preview instance
         this.options.responseInterceptor = options.responseInterceptor;
 
+        this.options.annotationsOptions = merge(options.annotationsOptions, {
```

Comment:
Can we update the host app to pass in the `features` option, instead of `showAnnotationsHighlightText`?

Reply:
Sure, I was trying to be backwards compatible, but we can just make a clean transition over... across 3 codebases 😄

Reply:
Okay. We have to make the change in all three anyway, don't we?